CHANGELOG.adoc

```diff
@@ -82,6 +82,7 @@ image:doc/screenshots/inside_outside_labels.png[Isinde outside label example, 70
 - https://github.com/eclipse-sirius/sirius-web/issues/3237[#3237] [diagram] Add a `DiagramServices` class containing services to expand and collapse nodes from AQL expressions (via the new `diagramServices` variable) and Java services.
 - https://github.com/eclipse-sirius/sirius-web/issues/3282[#3282] [diagram] Add diagram assertions and navigators to ease the definition of diagram tests.
 - https://github.com/eclipse-sirius/sirius-web/issues/3277[#3277] [gantt] Add support for dependency creation
+- https://github.com/eclipse-sirius/sirius-web/issues/3333[#3333] [sirius-web] Added IMigrationParticipant interface, contribute implemention to migrate models
```

Comment:
Since a part of your code does only depend on EMF, EMFJson and Sirius Components EMF, I'll move the relevant classes / interfaces in sirius-components-emf to make them as reusable as possible by downstream projects. I'll just move IMigrationParticipant, MigrationExtendedMetaData and MigrationJsonProcessor.